src/TibiaCharactersCharacterV3.go

```diff
@@ -217,9 +216,12 @@ func TibiaCharactersCharacterV3(c *gin.Context) {
 				regex1 := regexp.MustCompile(`<td.*class=.[a-zA-Z0-9_.-]+..*>(.*):<\/.*td><td>(.*)<\/td>`)
 				subma1 := regex1.FindAllStringSubmatch(CharacterTrHTML, -1)
 
-				if len(subma1) > 0 {
+				var AccountStatusString = string([]byte{65, 99, 99, 111, 117, 110, 116, 194, 160, 83, 116, 97, 116, 117, 115})
+				var GuildMembershipString = string([]byte{71, 117, 105, 108, 100, 194, 160, 77, 101, 109, 98, 101, 114, 115, 104, 105, 112})
```

Comment:
This was an interesting way of solving this 😮 
Maybe there is a way of using letters somehow to make it more readable?

Reply:
Pushed the fix for the underscore var.

But this is still a problem because of Tibia's terrible data hahaha
`194` is the issue here